src/main/frontend/components/right_sidebar.cljs

```diff
@@ -102,7 +102,7 @@
 
     :page-presentation
     (let [page-name (:block/name (db/entity db-id))]
-      [[:a {:href (rfe/href :page {:name page-name})}
+      [[:a.page-title {:href (rfe/href :page {:name page-name})}
```

Comment:
Accidentally close my [prev pull request](https://github.com/logseq/logseq/pull/7285) =(
So add this fix again